docs/public-networks/concepts/transactions/pool.md

```diff
@@ -53,6 +54,28 @@ If you previously configured transaction pool behavior, upgrade to the layered t
 You can opt out of the layered transaction pool implementation by setting the
 [`--tx-pool`](../../reference/cli/options.md#tx-pool) option to `sequenced`.
 
+### Penalize transient invalid pending transactions
+
+Transient invalid pending transactions cannot be included in the current block but might be included
```

Comment:
I think this is a bit of a run on. Would suggest

```suggestion
Transient invalid pending transactions cannot be included in the current block but might be included in a future one. This can happen due to issues like insufficient balance in the sender's wallet or a gas price below the minimum.
```